docs/user-manuals/cloneset.md

```diff
@@ -606,6 +612,9 @@ spec:
 
 ![Lifecycle circulation](/img/docs/user-manuals/cloneset-lifecycle.png)
 
+- When CloneSet create a Pod (including scale in and recreate update):
+  - The Pod will be regarded as `Available` and util the `PreNormal` hook is satified.
```

Comment:
The Pod will be be regarded as Available only after PreNormal hook is satisfied.